src/sentry/incidents/grouptype.py

```diff
@@ -25,3 +25,4 @@ class MetricAlertFire(GroupType):
     enable_auto_resolve = False
     enable_escalation_detection = False
     detector_handler = MetricAlertDetectorHandler
+    detector_validator = MetricAlertsDetectorValidator
```

Comment:
🤔 is there a way we could enforce the consistency here across all the GroupTypes for a detector?

Reply:
Not totally sure what you mean

Reply:
I'm just curious if we can say `detector_validator` will always be set on the group_type for MetricIssues? that way it's not like `validator` on one group type and `detector_validator` on another.